index.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module "@andrewlevada/send-beacon" {
```

Comment:
The module name should be "send-beacon"

Reply:
Yes, sorry, only meant to push first two commits